xbps-src

```diff
@@ -464,6 +464,8 @@ fi
 # Read settings from config file
 [ -s "$XBPS_CONFIG_FILE" ] && . $XBPS_CONFIG_FILE &>/dev/null
 
+[ -n "${XBPS_FORCE_COLORS}" ] && export -n NOCOLORS=''
```

Comment:
This mechanism is a bit wrong, because anyone who decides to enable it locally can't override it with the `-L` flag.

You should maybe check first if `NOCOLORS` hasn't been set.

Reply:
`NOCOLORS` is set used by both the tty check and the `NO_COLOR` check here:
https://github.com/void-linux/void-packages/blob/f7e694733df137bc5957d62a43bb81b27a5af130/xbps-src#L408-L413
and by the argument parsing just above it. The configuration is loaded much later than those 3 points, so detection of `-L`/`NO_COLOR` would be faked by the tty check.

Not sure what the best way to go is, would it be fine to move tty/NO_COLORS to where the override currently is, and change the var set by arg parsing to an intermediary var?

Another way would be to move the tty check to where the override is, where it could simply be skipped in the presence of the command. This might be the better way

Reply:
Since Github has changed the way they run CI, the whole override mechanism is not longer needed and has been removed from the PR.